src/Castle.Core/Castle.Core.csproj

```diff
@@ -43,6 +43,11 @@
 		<PackageReference Include="System.ComponentModel" Version="4.3.0" />
 		<PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
 	</ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)'!='net35'AND'$(TargetFramework)'!='net40'">
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
These packages should probably be defined with open upper version so application layer can have higher version installed